src/components/MapFacetContents.js

```diff
@@ -31,11 +31,20 @@ type MapFacetContentsProps = {
   size: any;
   /** The public key with which to connect to the mapbox public apis. */
   mapboxKey: string;
+  /** Custom location pointer glyphicon, default is the map-marker glyphicon */
+  glyph: string | null;
+  /** Custom location pointer image uri, if available image is shown instead of glyphicon */
+  uri: string | null;
+  /** Custom tooltip text to convey what the location pointers are for, for e.g. customers, transaction location etc */
+  tooltip: string;
```

Comment:
We should let people pass a string that is understood by the `StringUtils.fmt()` method and pass the count as the value parameter. (And then call that method later...)
This would allow the same map's tooltips to say things like "There are 4 stores here" and "There is only one store here" if the count is 4 or 1, respectively.
You could theoretically prepend the number like you're doing now if the string isn't a format string (i.e. if the format string before and after calling `StringUtild.fmt()` is the same...).
In any case, update the comment here to describe it in a bit more detail.

Reply:
That makes sense, I wasn't aware of that function in StringUtils. I changed the tooltips to use StringUtils.fmt() and updated the comment as well.